homeassistant/components/systemmonitor/coordinator.py

```diff
@@ -2,23 +2,26 @@
 from __future__ import annotations
 
 from abc import abstractmethod
+from collections import namedtuple
 from datetime import datetime
 import logging
 import os
 from typing import TypeVar
 
 import psutil
 from psutil._common import sdiskusage, shwtemp, snetio, snicaddr, sswap
-from psutil._pslinux import svmem
 
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_component import DEFAULT_SCAN_INTERVAL
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 from homeassistant.util import dt as dt_util
 
-from .const import DOMAIN
-
 _LOGGER = logging.getLogger(__name__)
+# psutil define virtual_memory by platform. Create our own definition here to be platform independent.
+virtual_memory = namedtuple(
```

Comment:
Please use the new style syntax:

https://docs.python.org/3/library/typing.html#typing.NamedTuple